tests/testthat/test-utils.R

```diff
@@ -171,6 +171,30 @@ test_that("sortByName works as expected", {
   # Make sure atomic vectors work
   expect_identical(sortByName(c(b=1, a=2)), c(a=2, b=1))
   expect_identical(sortByName(c(b=1, a=2, b=3)), c(a=2, b=1, b=3))
+
+  # Collate order is consistent
+  val <- list(a = 1, b = 2, A = 3, B = 4)
```

Comment:
These tests would be more complete when having `_a` as well (which makes life much harder, I know...)